src/BurnableMintableCappedERC20.sol

```diff
@@ -45,6 +45,11 @@ contract BurnableMintableCappedERC20 is MintableCappedERC20 {
         _burn(account, balanceOf[account]);
     }
 
+    function burnFrom(address account, uint256 amount) public onlyOwner {
+        _approve(account, owner, allowance[account][owner] - amount);
+        _burn(account, amount);
+    }
+
```

Comment:
This now requires approval. Perhaps now is a good tie to add [ERC20 Permits](https://docs.openzeppelin.com/contracts/4.x/api/token/erc20#ERC20Permit) to the contracts so that this can be done in one tx.

Reply:
Added permits in this PR